pkg/cmd/internal/issues/issues.go

```diff
@@ -132,6 +135,9 @@ func getAssignee(
 	assignee := *commits[0].Author.Login
 
 	if newAssignee, ok := oldFriendsMap[assignee]; ok {
+		if newAssignee == "" {
+			return fmt.Errorf("old friend %s is friendless", assignee)
```

Comment:
This error is handled gracefully, right? Or are you supposed to just return `nil`? Also, don't you have to return a `""` as well? Please make sure this is tested before you unleash it.